tools_considered_and_the_screening_decisions.md

```diff
@@ -0,0 +1,178 @@
+|DistillerSR                                                                    |         1|         1|         1|                                                                            |
+|EPPI-Reviewer                                                                  |         1|         1|         1|                                                                            |
+|FASTREAD                                                                       |         1|         0|         1|                                                                            |
+|Rayyan                                                                         |         1|         1|         1|                                                                            |
+|RobotAnalyst                                                                   |         1|         0|         1|                                                                            |
+|SWIFT-Active Screener                                                          |         1|         1|         1|                                                                            |
+|Active_learning_document_screening                                             |         1|         0|         0|no software                                                                 |
+|Active-learning-for-systematic-review                                          |         1|         0|         0|no software                                                                 |
+|AntConc                                                                        |         0|         1|         0|no AL or ML for supporting literature screening (also last edit 2011)       |
+|ASReview-covid                                                                 |         1|         0|         0|limited to COVID19                                                          |
+|Bibot                                                                          |         1|         0|         0|no RITL-based AL cycle                                                      |
+|BioReader (Biomedical Research Article Distiller)                              |         0|         1|         0|classification and not AL                                                   |
+|Buhos                                                                          |         1|         1|         0|no AL or ML for supporting literature screening                             |
+|Cadima                                                                         |         1|         1|         0|no RITL-based AL cycle                                                      |
+|Cientopolis Scolr                                                              |         0|         1|         0|website is down and no publications                                         |
```

Comment:
well, not necessarily so, but since the website was down, we searched for a paper with maybe a link to a GitHub repository or anything. But we could not find anything